cypress/integration/artAppraisals.spec.ts

```diff
@@ -0,0 +1,6 @@
+describe("/art-appraisals", () => {
+  it("renders the page", () => {
+    cy.visit("art-appraisals")
+    cy.contains("Art Appraisals")
```

Comment:
**question (non-blocking):** When does it make sense to use Cypress in Force rather than adding to Integrity?

Secondarily, does it make more sense to use RTL for this since it also functions like an integration test but is more lightweight?